package.json

```diff
@@ -25,22 +25,21 @@
     "express": "3.11.0",
     "lodash": "^4.17.13",
     "logger-sharelatex": "^1.7.0",
-    "lynx": "0.0.11",
     "metrics-sharelatex": "^2.2.0",
     "mongojs": "^2.6.0",
     "redis-sharelatex": "^1.0.11",
-    "request": "2.47.0",
+    "request": "^2.47.0",
     "requestretry": "^1.12.0",
```

Comment:
This package pulls in an old version of the `request` package - essentially ignoring your bump.

See the diff from moving to 3.x https://github.com/das7pad/document-updater-sharelatex/commit/a66626c03deef96278a309747220fb5fc71f41b9

From the change log https://github.com/FGRibreau/node-request-retry/blob/master/CHANGELOG.md:
- 2.0 `request` as `peerDependecy`
- 3.0 retry on status code 429 
- 4.0 signature change for `retryStrategy` (which we do not use)